src/manage-apps/using-appimages.md

```diff
@@ -0,0 +1,53 @@
+# AppImages
+
+The AppImage format packages an application in a single, self-contained file. This format allows for a simplified installation process. Appimage.github.io provides links to download hundreds of applications as AppImages. Once downloaded, an AppImage must be given execute permissions in order to launch.
+
+---
+
+## Downloading an AppImage
+
+1. Navigate to [appimage.github.io](https://appimage.github.io/) and click the `Download` button next to an application.
+
+    ![Click Download](/images/using-appimages/click-download.png)
+
+2. The link may take you to the application's github repository. Download the latest stable version of the AppImage. Make sure you download the compatible version for your 32-bit (i386) or 64-bit (x86-64) CPU architecture. All modern personal computers will have a 64-bit CPU.
+
+    ![Download AppImage](/images/using-appimages/download-appimage.png)
+
+3. Navigate to the downloaded AppImage. Right-click the AppImage file and choose `Properties`.
+
+    ![Choose Properties](/images/using-appimages/choose-properties.png)
+
+4. Select the `Permissions` tab and check the box for `Execute`.
+
+    ![Execute Permissions](/images/using-appimages/execute-permissions.png)
+
```

Comment:
You didn't explain how to launch the file. I would include a step to double-click the file, just for completeness' sake.